### PR TITLE
Added missed phrases to RU locale (got from en locale)

### PR DESCRIPTION
**Pull request by zlocate**
Added missed phrases from EN locale.

### CODE REVIEW

**Review by whymarrh — CHANGES_REQUESTED** [review]
Hey @zlocate, thanks a bunch of helping with translations! The JSON file currently has a few errors in its formatting—a tool/site like https://jsonlint.com can help if you want to have a go at fixing those up first.

(cc @MetaMask/translators any of y'all able to review this? 😅)

**Comment by zlocate** [comment]
@whymarrh thank you for feedback. Some phrases adopted at Russian common grammar rules but i would like external review from native speakers (i'm native Russian speaker).
Sorry for breaking indent in source file. I will fix it asap

**Comment by zlocate** [comment]
@whymarrh fixed. Please review current version.

**Comment by nujabes403** [comment]
@zlocate Thank you for your effort!

**Review by whymarrh — DISMISSED** [review]
Thank you @zlocate! ❤️

File: app/_locales/ru/messages.json [hunk 1]
```diff
@@ -931,5 +931,998 @@
...
+    "message": "из"
+  },
+  "onlySendTokensToAccountAddress": {
+    "message": "Отправляйте $1 на адрес учетной записи Ethereum."
+  },
+  "openInTab": {
+    "message": "Открыть во вкладке"
+  },
+  "orderOneHere": {
+    "message": "Закажите Trezor или Ledger и храните свои средства в холодильнике"
```
Comment (dryajov) [inline app/_locales/ru/messages.json]: Закажите Trezor или Ledger и храните свои средства в холодном кошельке

File: app/_locales/ru/messages.json [hunk 2]
```diff
@@ -931,5 +931,998 @@
...
+    "message": "Просмотр информации о сети"
+  },
+  "welcomeBack": {
+    "message": "Рад видеть вас снова!"
+  },
+  "writePhrase": {
+    "message": "Напишите эту фразу на листе бумаги и храните в надежном месте. Если вы хотите еще большей безопасности, запишите это на нескольких листах бумаги и храните каждый в 2 - 3 разных местах."
+  },
+  "yesLetsTry": {
+    "message": "Да, давайте начнем"
```
Comment (dryajov) [inline app/_locales/ru/messages.json]: Maybe - "Да, давайте попробуем" ?

**Review by dryajov — CHANGES_REQUESTED** [review]
Just a few comments/suggestions. 

Thanks for contributing.

**Review by whymarrh — APPROVED** [review]
Thanks again @zlocate!

**Pull request merged**